package.json

```diff
@@ -32,35 +32,35 @@
     "babel-eslint": "^10.0.2",
     "babel-jest": "^24.8.0",
     "babel-loader": "^8.0.2",
-    "copy-webpack-plugin": "^4.5.1",
-    "core-js": "^3.1.4",
-    "crowdin-api": "^2.0.2",
-    "css-loader": "^1.0.0",
+    "copy-webpack-plugin": "^5.0.0",
+    "core-js": "3",
+    "crowdin-api": "^4.0.0",
+    "css-loader": "^3.0.0",
     "eslint": "^6.1.0",
-    "eslint-plugin-flowtype": "^4.2.0",
+    "eslint-plugin-flowtype": "^4.0.0",
     "eslint-plugin-react": "^7.14.3",
     "fancy-log": "^1.3.3",
-    "file-loader": "^2.0.0",
-    "flow-bin": "^0.105.2",
+    "file-loader": "^5.0.0",
+    "flow-bin": "^0.114.0",
```

Comment:
I never integrated Flow with CI. Could you try running `yarn flow` after this upgrade and make sure the repo still typechecks? There may be other updates required.

Reply:
```
~/Prog/tabwrangler(updates-try2 ✔) yarn flow
yarn run v1.21.1
$ /Users/ktamas/Prog/tabwrangler/node_modules/.bin/flow
Launching Flow server for /Users/ktamas/Prog/tabwrangler
Spawned flow server (pid=10477)
Logs will go to /private/tmp/flow/zSUserszSktamaszSProgzStabwrangler.log
Monitor logs will go to /private/tmp/flow/zSUserszSktamaszSProgzStabwrangler.monitor_log
No errors!
✨  Done in 5.68s.
```